testutil/testutil.go

```diff
@@ -186,6 +186,11 @@ func GetTestRMQCreds(t *testing.T) (string, string, string) {
 	return v[0], v[1], v[2]
 }
 
+func GetTestMDBACreds(t *testing.T) (string, string) {
+	v := SkipTestEnvUnset(t, "PRIVATE_KEY", "PUBLIC_KEY")
```

Comment:
Could we have a prefix to the env variables, something like `MONGODB_ATLAS_PRIVATE_KEY`? Although it's longer I'd rather opt on the side of being over explicit 😅

Reply:
Makes sense. Will do
Thank you!